src/importer/ManageApi.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Threading.Tasks;
+using GovUk.Education.ManageCourses.ApiClient;
+
+namespace GovUk.Education.ManageCourses.UcasCourseImporter
+{
+    public class ManageApi 
+    {
+        private readonly string apiLocation;
+        private readonly string bearerToken;
+
+        public ManageApi(string apiLocation, string bearerToken)
+        {
+            this.apiLocation = apiLocation;
+            this.bearerToken = bearerToken;
+        }
+        public void PostPayload(UcasPayload payload)
+        {
+            Console.WriteLine("Posting to api...");
+            var client = new ManageCoursesApiClient(new ApiConf(bearerToken), new System.Net.Http.HttpClient());  
```

Comment:
you may as well have
`private readonly string client;`
ie Constructor takes the string and sets the client

Reply:
❓

Reply:
```c#
private readonly ManageCoursesApiClient client;
public ManageApi(api, url) {
  client= new ManageCoursesApiClient(api, url);
}
```

Reply:
aaah!